src/scala/tiefvision-web/app/controllers/TiefVisionResourcesAssets.scala

```diff
@@ -5,11 +5,24 @@
   */
 package controllers
 
+import java.io.File
 import com.google.inject.Inject
-import play.api.{Play, Environment}
+import play.api.{Environment, Logger, Play}
+import play.api.mvc._
 
-class TiefVisionResourcesAssets @Inject()(environment: Environment) extends ExternalAssets {
+class TiefVisionResourcesAssets @Inject()(environment: Environment) extends Controller {
 
-  def atResources(rootPath: String, file: String) = super.at(s"${Configuration.HomeFolder}/$rootPath", file)
+  // Request local resources within the TIEFVISION_HOME folder (for security reasons)
+  def atResources(rootPath: String, filePath: String) = Action {
+    val file = new File(s"${Configuration.HomeFolder}/$rootPath/$filePath")
```

Comment:
I doubt this will be moved to windows, but if that is something you want to aim for, we can